src/main/java/nl/hsac/fitnesse/symbols/RandomString.java

```diff
@@ -41,13 +41,56 @@ public RandomString() {
     }
 
     public String toTarget(Translator translator, Symbol symbol) {
+        String param = symbol.getProperty(MIN_MAX_LENGTH, null);
         String permitted = symbol.getProperty(CHARACTERS, DEFAULT_CHARS);
-        int length = getProperty(symbol, LENGTH, -1);
         String prefix = symbol.getProperty(PREFIX, "");
-        length = length - prefix.length();
-        if (length < 1) {
-            length = RANDOM_UTIL.random(100);
-        }
+        int length = getRandomStringLength(param, prefix);
         return prefix + RANDOM_UTIL.randomString(permitted, length);
     }
+
+    public int getRandomStringLength(String param, String prefix) {
+        int randomStringLength;
+        int minimalLength;
+        int maximalLength;
+        int prefixLength;
+
+        //Handle the prefix input parameter
+        if (prefix == "") {
```

Comment:
The correct way to do this is: `"".equals(prefix)`, but why is an `if` needed? The length of `""` is 0, so the 'else' block would yield same result?
In other words just replace entire if/then/else with:
`prefixLength = prefix.length();`, or am I overlooking something?